spec/controllers/posts_controller_spec.rb

```diff
@@ -89,5 +90,39 @@
       end
     end
 
+    describe "#create" do
```

Comment:
looks great! it does look like a lot, but it would be great to break down and walk through with students.  Maybe include the bitly gem, to leave a hint for students?  Should we merge the pull request?